site_root/input/resources/profiles/StructureDefinition-kfdrc-no-phi-phenotype.json

```diff
@@ -0,0 +1,114 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "kfdrc-no-phi-phenotype",
+  "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-no-phi-phenotype",
+  "version": "0.1.0",
+  "name": "kfdrc-phenotype",
+  "title": "Kids First DRC Phenotype without Protected Health Information",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "Observation",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Observation",
```

Comment:
The no phi profiles should be based on the base kfdrc profile, not the base FHIR profile. The type is still Observation

Reply:
Gotcha! I will update all the no PHI profiles' base definitions with PHI ones!

Reply:
Updated and validated!